lib/active_record_shards.rb

```diff
@@ -21,6 +22,9 @@ def self.rails_env
 ActiveRecord::Base.extend(ActiveRecordShards::ConfigurationParser)
 ActiveRecord::Base.extend(ActiveRecordShards::Model)
 ActiveRecord::Base.extend(ActiveRecordShards::ConnectionSwitcher)
+class << ActiveRecord::Base
+  prepend ActiveRecordShards::ConnectionSchemaCache
+end
```

Comment:
It looks like the same patch being applied in two different places (`ConnectionSwitcher` and `ConnectionSchemaCache`) – but both places are actually the same place: `ActiveRecord::Base` in class scope. Am I mistaken?

Reply:
no, but wanted to use `prepend` to not add another alias_method